shub/config.py

```diff
@@ -277,13 +277,20 @@ def get_version(self):
             return str(self.version)
 
     def get_target_conf(self, target, auth_required=True):
+        class Apikey(object):
+            def __init__(self, apikey=''):
+                self.apikey = apikey
+
+            def __repr__(self):
+                return 'XXXXXXXXXXX'
```

Comment:
I wonder if it would make sense to, instead of completely hiding the API key, show only a few characters at the end or at the beginning, so that a person that knows the whole key can verify that the right one is being picked.

Reply:
Sounds good to me, +1 to keep a few characters at the beginning